spec/models/sales_log_spec.rb

```diff
@@ -538,6 +538,11 @@
   describe "derived variables" do
     let(:sales_log) { create(:sales_log, :completed) }
 
+    before do
+      Timecop.return
+      Singleton.__init__(FormHandler)
+    end
```

Comment:
Currently there is a time fix on the entire test file. That's too big to sort in this ticket, hence I've just taken this section.